client/src/containers/backend/Content/Features/Detail.js

```diff
@@ -200,38 +201,47 @@ class FeatureDetailContainer extends PureComponent {
   render() {
     const feature = this.feature(this.props);
     const isNew = this.isNew(this.props);
+    const action = isNew ? "create" : "update";
     const previewFeature = this.previewableFeature(this.props);
     return (
-      <div>
-        {this.state.confirmation ? (
-          <Dialog.Confirm {...this.state.confirmation} />
-        ) : null}
-        {isNew ? this.newHeader() : this.featureHeader(feature)}
-        <section className="backend-panel">
-          <div className="container">
-            {feature || isNew ? (
-              <div className="panel">
-                <section>
-                  {previewFeature ? (
-                    <div className="form-secondary">
-                      <div className="form-input">
-                        <label>Feature Preview</label>
-                        <span className="instructions">
-                          This is an approximate preview of your feature.
-                          Foreground, background, and markdown will not be
-                          displayed until the feature is saved.
-                        </span>
-                        <Layout.Splash feature={previewFeature} preview />
+      <HigherOrder.Authorize
+        entity="feature"
+        failureFatalError={{
+          detail: `You are not allowed to ${action} features.`
+        }}
+        ability={action}
```

Comment:
The action here can be either `update` or `create`. If we're updating, we should authorize against the instance, not the class ability. Something like this:

```
const authProps = isNew ? { entity: feature, action: "update" } : { entity: "feature", action: "create" }
return (
  <HigherOrder.Authorize {...authProps} otherProp="blah" />
```